buildSrc/src/main/java/com/uber/okbuck/composer/jvm/JvmLibraryRuleComposer.java

```diff
@@ -10,6 +10,7 @@
 import java.util.List;
 import java.util.Set;
 
+@SuppressWarnings("NullAway")
```

Comment:
Why is this related?

Reply:
not sure, nullaway threw errors on
```
          new JvmBinaryRule()
              .mainClassName(target.getMainClass())
```
target.getMainClass() could be null

not sure why it isn't wasn't caught before